common/persistence/dataManagerInterfaces.go

```diff
@@ -343,6 +343,7 @@ type (
 		AutoResetPoints                    *types.ResetPoints
 		Memo                               map[string][]byte
 		SearchAttributes                   map[string][]byte
+		PartitionConfig                    map[string]string
```

Comment:
I was considering type-aliasing the PartitionConfig just to make any refactoring easier, but this is also fine imho